spec/_attachments/ic.did

```diff
@@ -184,6 +192,14 @@ service ic : {
   bitcoin_send_transaction: (send_transaction_request) -> ();
   bitcoin_get_current_fee_percentiles: (get_current_fee_percentiles_request) -> (vec millisatoshi_per_byte);
 
+  // metrics interface
+  metrics : (record {
```

Comment:
Should we make this more specific, since there may be canister (or query etc) metrics in the future?
Or do you feel there is no need for this?
```suggestion
  subnet_metrics : (record {
```

Reply:
Let's go for node_metrics everywhere. Originally, I've drafted the type `metrics` as a variant type so that other kinds of metrics can be added. But they'll likely have a different interface, e.g., you'd also specify a requested canister ID, and thus a separate method seems more appropriate (with corresponding types).

Reply:
```suggestion
  node_metrics : (record {
```